mmseg/datasets/custom.py

```diff
@@ -286,9 +286,13 @@ def pre_eval(self, preds, indices):
         for pred, index in zip(preds, indices):
             seg_map = self.get_gt_seg_map_by_idx(index)
             pre_eval_results.append(
-                intersect_and_union(pred, seg_map, len(self.CLASSES),
-                                    self.ignore_index, self.label_map,
-                                    self.reduce_zero_label))
+                intersect_and_union(
+                    pred,
+                    seg_map,
+                    len(self.CLASSES),
+                    self.ignore_index,
+                    label_map=dict(),
```

Comment:
Might add some comments for this revise, like
```suggestion
                    # as the labels has been converted when dataset initialized
                    # in `get_palette_for_custom_classes ` this `label_map`
                    # should be `dict()`, see https://github.com/open-mmlab/mmsegmentation/issues/1415
                    # for more ditails 
                    label_map=dict(),
```